doc/dev/release-checklists.rst

```diff
@@ -37,21 +37,19 @@ Make sure X (and, ideally, X+1) is defined:
 - [x] src/common/ceph_strings.cc (`ceph_release_name()`)
 - [x] src/include/rados.h (`CEPH_RELEASE_*` and `MAX`)
 - [x] src/include/rbd/librbd.h (`LIBRBD_VER_MINOR` to X)
-- [x] src/mon/mon_types.h (`ceph::features::mon::FEATURE_*` and related structs and helpers; note that monmaptool CLI test output will need adjustment)
+- [x] src/mon/mon_types.h (`ceph::features::mon::FEATURE_*` and related structs and helpers)
 - [x] src/mds/cephfs_features.h (`CEPHFS_CURRENT_RELEASE`)
 
 Scripts
 ~~~~~~~
 
 - [x] src/script/backport-resolve-issue (`releases()`, `ver_to_release()`... but for X-1)
-- [x] src/script/ceph-release-notes (X-1)
-- [ ] ceph-build.git scripts/build_utils.sh `release_from_version()`
+- [ ] src/script/ceph-release-notes (X-1)
```

Comment:
@neha-ojha @jdurgin @rzarzynski do you know what this is for? I don't see anything to change.

Reply:
IIRC, it is just a matter of cleaning up old releases and adding new ones in src/script/ceph-release-notes

Reply:
:scream: